base/client.jl

```diff
@@ -71,6 +71,18 @@ function repl_cmd(cmd, out)
         ENV["OLDPWD"] = new_oldpwd
         println(out, pwd())
     else
+        @windows_only begin
```

Comment:
This logic would be best extracted into a (platform agnostic) function called cmdshell_escape, so it can be reused